# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1202874148 18000
#      Tue Feb 12 22:42:28 2008 -0500
# Branch release-3-0-x
# Node ID 128a1ef5e21e1c95d8de15224ed03ccc9cf3d557
# Parent  1202614e8a735bd5f1a75f9e65be77997e4a708d
undo rest of botched sort patch

diff --git a/src/ov-range.h b/src/ov-range.h
--- a/src/ov-range.h
+++ b/src/ov-range.h
@@ -127,22 +127,16 @@ public:
   bool is_constant (void) const { return true; }
 
   bool is_range (void) const { return true; }
 
   octave_value all (int dim = 0) const;
 
   octave_value any (int dim = 0) const;
 
-  octave_value sort (octave_idx_type dim = 0, sortmode mode = UNDEFINED) const
-    { return array_value ().sort (dim, mode); }
-  octave_value sort (Array<octave_idx_type> &sidx, octave_idx_type dim = 0,
-		     sortmode mode = UNDEFINED) const
-    { return array_value ().sort (sidx, dim, mode); }
-
   bool is_real_type (void) const { return true; }
 
   bool is_double_type (void) const { return true; }
 
   bool valid_as_scalar_index (void) const
     {
       double b = range.base ();
       return (range.nelem () == 1
